src/sdk/model/vapp/vapp-copy-move-request.ts

```diff
@@ -0,0 +1,81 @@
+import { VappCopyMoveRequestJson } from './__json__/vapp-copy-move-request-json';
+import { VappChildVmCopyMoveRequestJson } from './__json__/vapp-child-vm-copy-move-request-json';
+
+/**
+ * vApp copy move request.
+ */
+/* istanbul ignore next: autogenerated */
+export class VappCopyMoveRequest {
+
+  private readonly _json: VappCopyMoveRequestJson;
+
+  constructor(vappCopyMoveRequest: VappCopyMoveRequest);
+  constructor(vappCopyMoveRequestJson: VappCopyMoveRequestJson);
+  constructor(name: string, description: string, vmSpecs: Array<VappChildVmCopyMoveRequestJson>, vdcUuid: string);
+  constructor(firstParam: string | VappCopyMoveRequest | VappCopyMoveRequestJson, description?: string,
+              vmSpecs?: Array<VappChildVmCopyMoveRequestJson>, vdcUuid?: string) {
+    if (typeof firstParam === 'string') {
+      // Parameters constructor
+      this._json = {
+        name: firstParam,
+        description: description,
+        vm_specs: vmSpecs,
+        vdc_uuid: vdcUuid
+      } as VappCopyMoveRequestJson;
+    } else if (firstParam instanceof VappCopyMoveRequest) {
+      // Copy constructor
+      this._json = (firstParam as VappCopyMoveRequest).json;
+    } else {
+      // Json or empty constructor
+      this._json = (firstParam || {}) as VappCopyMoveRequestJson;
+    }
+  }
+
+  /**
+   * Get name.
+   * @returns {string}
+   */
+  get name(): string {
+    return this._json.name;
+  }
+
+  /**
+   * Get description.
+   * @returns {string}
+   */
+  get description(): string {
+    return this._json.description;
+  }
+
+  /**
+   * Get vm specs.
+   * @returns {Array<VappChildVmCopyMoveRequestJson>}
+   */
+  get vmSpecs(): Array<VappChildVmCopyMoveRequestJson> {
```

Comment:
should return an instance of the wrapper class, not the JSON interface

Reply:
@cfsnyder fixed